framework_tvb/tvb/interfaces/rest/client/operation/operation_api.py

```diff
@@ -38,21 +38,21 @@
 
 class OperationApi(MainApi):
     @handle_response
-    def get_operation_status(self, operation_gid):
+    def get_operation_status(self, operation_gid, token):
         return requests.get(self.build_request_url(RestLink.OPERATION_STATUS.compute_url(True, {
             LinkPlaceholder.OPERATION_GID.value: operation_gid
-        })))
+        })), headers=self.get_headers(token))
```

Comment:
Could we have a base method for building the request in MainAPI, and adding the header only once ?